library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/impl/ResourceLoaderImpl.java

```diff
@@ -523,4 +536,36 @@ public static void appendLanguageEntries(@NotNull Map<String, String> map) {
 			}
 		}
 	}
+
+	/* Static pack stuff */
+	private static List<ResourcePack> findUserStaticPacks() {
+		List<ResourcePack> returnList = new ArrayList<>();
+		File directoryFile = QuiltLoader.getGameDir().resolve(STATIC_PACK_ROOT).toFile();
+		File[] potentialPackFiles = directoryFile.listFiles(filterFile -> {
```

Comment:
Given that most things use the NIO Path API, it might make more sense to use the `Files` NIO API to search packs.

Reply:
Tested implementing it that way, and it both works and is slightly cleaner overall so it'll be swapped to using `Files.walk` in the next commit where `listFiles` is currently.